src/LogAnalysis/Types.hs

```diff
@@ -48,6 +56,20 @@ data Knowledge = Knowledge
     -- ^ The FAQ number that will be displayed on the official Cardano FAQ page
     }
 
+-- | File format
+data FileFormat
```

Comment:
`(LogFile lfContent * FileFormat) + (LogFile lfContent * FileFormat)` == `(LogFile lfContent + LogFile lfContent) * FileFormat` == `(LogFile lfContent * (1 + 1)) * FileFormat` == `(2 * LogFile lfContent) * FileFormat` == `2 * (LogFile lfContent * FileFormat)`

Reply:
Will fix/simplify for example.